frontend/app.js

```diff
@@ -1060,8 +1073,10 @@ function displayModal(message) {
 
     if (message.indexOf("Success!") > -1){
         $("#IDEButton").show();
+        $("#FolderButton").show();
     } else {
         $("#IDEButton").hide();
+        $("#FolderButton").show();
```

Comment:
should this be hide?

Reply:
if it fails to build, allow to direct to open folder as well maybe need to check files etc